commit 49345c8039fdcae0501663b524b59af39f726ed0
Author: David Ahern <dsa@cumulusnetworks.com>
Date:   Fri Mar 10 09:46:15 2017 -0800

    mpls: Send route delete notifications when router module is unloaded
    
    [ Upstream commit e37791ec1ad785b59022ae211f63a16189bacebf ]
    
    When the mpls_router module is unloaded, mpls routes are deleted but
    notifications are not sent to userspace leaving userspace caches
    out of sync. Add the call to mpls_notify_route in mpls_net_exit as
    routes are freed.
    
    Fixes: 0189197f44160 ("mpls: Basic routing support")
    Signed-off-by: David Ahern <dsa@cumulusnetworks.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1400_49345c8039fdcae0501663b524b59af39f726ed0.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1400_49345c8039fdcae0501663b524b59af39f726ed0.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
